### ISSUE TITLE
Nothing here

### ISSUE THREAD

**Issue by carrolltondatago** [GPUOpen-LibrariesAndSDKs/OCL-SDK#2]
There is nothing here beyond the license file.  Where is the SDK??

**Comment by BenjaminCoquelle** [comment]
In the release link. It is just the lib and headers to compile on windows
https://github.com/GPUOpen-LibrariesAndSDKs/OCL-SDK/releases

**Comment by herrBez** [comment]
I did not manage to find the software as well (http://developer.amd.com/tools-and-sdks/opencl-zone/) redirect to this repository for the OpenCL SDK. 


In the answer you mention only windows users. Is there also a repository for the linux users.

**Comment by herrBez** [comment]
I did not manage to find the software as well.. The link http://developer.amd.com/tools-and-sdks/opencl-zone/ redirects to this repository for the OpenCL SDK.

In the answer you mentioned only windows users. Is there also a repository for the linux users?

**Comment by BenjaminCoquelle** [comment]
You don't need a SDK for linux, as the library to link against is shipped with the driver.
The headers to compile can be found wih your package manager. 
For example on debian based distro
sudo apt-get install opencl-headers

**Comment by giuseppe-ciaccio** [comment]
The libOpenCL.so for Linux is NOT shipped along with the driver.  The complete SDK can be downloaded here:

http://developer.amd.com/amd-accelerated-parallel-processing-app-sdk/

(apparently not linked by any other webpage in the site)

In that page, the link to " MD5 Checksums, System Requirements & Driver Compatibility page" is broken.  The correct link is:

http://developer.amd.com/amd-accelerated-parallel-processing-app-sdk/system-requirements-driver-compatibility

In the latter you find the match between the SDK version, your OS version, and the required driver for the videocard.  Note that these drivers are not Free Software so they might not load with a Linux kernel.  Moreover, they might not work at all with current Linux 4.x kernels.  I personally had to install a Linux 3.x kernel and tweak the driver source code (in firegl_public.c) so as to make it appear as a GNU GPL driver.

**Comment by BenjaminCoquelle** [comment]
Yes it is if you work with amdgpu-pro stack. So if you download the latest driver from our website.
If you work with our old catalyst stack (that we don't support anymore). It is true that you don't have library.

**Comment by giuseppe-ciaccio** [comment]
Excellent, but if I try to find it, I'm directed here:
https://support.amd.com/en-us/download
then here:
https://support.amd.com/en-us/download/linux
and then, since I have a R9 390, I've different options depeding on the Linux distro.  For some of them, you are pointed to the amdgpu-pro.  For the others, you are directed to the catalyst.  All this is quite misleading :) But thank you for the information.

**Comment by BenjaminCoquelle** [comment]
OK got it. amdgpu-pro supports only the newer distribution. For older ones, only catalyst can be used. We will slowly remove all references to catalyst. For now we keep the links in case some developers need to work on older distribution. Unfortunately we don't provide updates anymore for these distro

So if possible could your work with ubuntu 16.04, RHEL/CentOS7.3 or any equivalent distribution

**Comment by vitorvas** [comment]
Unfortunately, looks like the new driver (and neither the SDK) provided by AMD didn't properly install OpenCL libraries/include files. I'm on CentOS 7.3 with RX 550 and still struggling.

**Comment by BenjaminCoquelle** [comment]
The header should be queried using your pacakge installer. So for CentOS 
"yum install opencl-headers".
The library is installed by the driver, look in /opt/amdgpu-pro folder

Thanks

**Comment by LWisteria** [comment]
Fact: OpenCL (runtime header cl.h) has its **version** (1.2, 2.0, 2.1 etc). Therefore, we should use same version's header as SDK's version to develop OpenCL software safely. Am I wrong?

According above condition, we cannot use package-provided "opencl-headers" obediently because its version could be different from SDK's.

I'd like to request you to provide the "cl.h" corresponding to amdgpu-pro's each version. Otherwise, could you tell us good place to know correct cl.h's version that we should/could use for each amdgpu-pro's version?

**Comment by Spudz76** [comment]
amdgpu-pro 17.xx and previous did not include any headers, you are required to locate and install (AMD removed it) AMD APP SDK 3.0 for the headers, and you will have two libraries but in general use the library from the driver not the SDK.

amdgpu-pro 18.xx just install the driver and then also package `opencl-amdgpu-pro-dev` for the headers, and the library comes with the driver.
